app/views/collections/show.html.erb

```diff
@@ -1,35 +1,32 @@
 <% provide :page_title, construct_page_title(@presenter.title) %>
-  <div itemscope itemtype="http://schema.org/CollectionPage" class="row">
-    <div class="col-sm-10 pull-right">
-      <header>
-        <%= render 'collection_title', presenter: @presenter %>
-      </header>
-      <%= render 'collection_description', presenter: @presenter %>
+<div itemscope itemtype="http://schema.org/CollectionPage" class="row">
+  <div class="col-sm-10 pull-right">
+    <header>
+      <%= render 'collection_title', presenter: @presenter %>
+    </header>
+    <%= render 'collection_description', presenter: @presenter %>
 
-      <% unless has_collection_search_parameters? %>
+    <% unless has_collection_search_parameters? %>
       <%= render 'collections/show_descriptions' %>
-      <% end %>
-    </div>
-    <div class="col-sm-2">
-      <%= render 'collections/media_display' %>
-      <% unless has_collection_search_parameters? %>
-      <%= render partial: 'collections/show_actions' %>
-      <% end %>
-    </div>
+    <% end %>
+  </div>
+  <div class="col-sm-2">
+    <%= render 'collections/media_display', presenter: @presenter %>
+    <% unless has_collection_search_parameters? %>
+      <%= render 'collections/show_actions', presenter: @presenter %>
+    <% end %>
   </div>
-
-<% if has_collection_search_parameters? %>
-    <% header_title = "Search Results within this Collection" %>
-<% else %>
-    <% header_title = "Items in this Collection" %>
-<% end %>
-
-<div style="float: right">
-<%= link_to t('sufia.collection.select_works'), sufia.dashboard_works_path %> | <%= link_to t('sufia.collection.add_work'), new_curation_concerns_generic_work_path %>
```

Comment:
We're dropping these links because we already have buttons elsewhere for selecting works and adding new works?